### PR TITLE
Remove banking stage forwarder counters

### PR DESCRIPTION
**Pull request by steviez**
#### Problem
The counters are either unused or have the data covered by a different datapoint

#### Summary of Changes
Rip em out

### CODE REVIEW

File: core/src/banking_stage/forwarder.rs [hunk 1]
```diff
@@ -228,11 +228,13 @@ impl Forwarder {
         usize,
         Option<Pubkey>,
     ) {
-        let (res, num_packets, forward_us, leader_pubkey) =
+        let (res, num_packets, _forward_us, leader_pubkey) =
             self.forward_packets(forward_option, forwardable_packets);
+        if let Err(ref err) = res {
+            warn!("failed to forward packets: {err}");
+        }
```
Comment (steviez) [inline core/src/banking_stage/forwarder.rs:235]: This is no longer inside `if num_packet > 0`. Currently, we just return `Ok()` if `forwardable_packets` is empty (and thus return 0 for num packets forwarded). Seems slightly more defensive to have it outside, but I don't feel to strongly and shift it back inside if desired

File: core/src/banking_stage/forwarder.rs [hunk 2]
```diff
@@ -228,11 +228,13 @@ impl Forwarder {
...
 
         if num_packets > 0 {
-            inc_new_counter_info!("banking_stage-forwarded_packets", num_packets);
```
Comment (steviez) [inline core/src/banking_stage/forwarder.rs:235]: This covered by `banking_stage_stats.forwarded_vote_count` and `banking_stage.forwarded_transaction_count`

File: core/src/banking_stage/forwarder.rs [hunk 3]
```diff
@@ -242,12 +244,6 @@ impl Forwarder {
                     .forwarded_transaction_count
                     .fetch_add(num_packets, Ordering::Relaxed);
             }
-
-            inc_new_counter_info!("banking_stage-forward-us", forward_us as usize, 1000, 1000);
```
Comment (steviez) [inline core/src/banking_stage/forwarder.rs:246]: I was unsure if this one is used at all. I could easily tie it into `banking_stage_stats`, but figured I'd check to see if you ever look at or could see a desire to look at this metric

Comment (apfitzge) [reply core/src/banking_stage/forwarder.rs:246]: nope never look at counters - they're useless

**Review by apfitzge — APPROVED** [review]

**Pull request merged**